docs/TextShaping.md

```diff
@@ -0,0 +1,84 @@
+# Text Shaping #
```

Comment:
This file should be added to `mkdocs.yml` in order to be included to the final HTML documentation